docs/pages/eas/metadata/config.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Create config files for EAS Metadata
+sidebar_title: Creating the store config
+---
+
+import { Callout } from '~/ui/components/Callout';
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+EAS Metadata is designed to be as simple as possible while still being extensible to cover almost all use cases.
+By default, it uses a simple **store.config.json** file located at the _root of your project_.
+
+You can configure the path or name of the store config file with the **eas.json** [`metadataPath`](../../submit/eas-json.md#metadatapath) property.
+Besides the default JSON format, EAS Metadata also supports more dynamic config using JavaScript files.
+
+## Static store config
+
+The default store config type for EAS Metadata is a simple JSON file.
+The example below shows an example store config with basic App Store information written in English (U.S.).
+
+You can find all configuration options in the [store config schema](./schema.md).
+
+> If you have the [VS Code Expo plugin](https://github.com/expo/vscode-expo#readme) installed, you get auto-complete, suggestions, and warnings for **store.config.json** files.
+
+<CodeBlocksTable tabs={['store.config.json']}>
+
+```json
+{
+  "configVersion": 0,
+  "apple": {
+    "info": {
+      "en-US": {
+        "title": "Awesome App",
+        "subtitle": "Your self-made awesome app",
+        "description": "The most awesome app you've ever seen",
+        "keywords": ["awesome", "app"],
+        "marketingUrl": "https://example.com/en/promo",
+        "supportUrl": "https://example.com/en/support",
+        "privacyPolicyUrl": "https://example.com/en/privacy"
+      }
+    }
+  }
+}
+```
+
+</CodeBlocksTable>
+
+
+## Dynamic store config
+
+The static config can get you far, but sometimes it's better to use a dynamic value.
+For example, copyright notices usually contain the current year, which is easy to forget to update every year.
+You can make this dynamic by using a **store.config.js** file configured with the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in your **eas.json** file.
```

Comment:
```suggestion
Metadata like the **copyright notice** should contain the current year, with EAS Metadata this can be automated.

Generate content dynamically by creating a JavaScript config file **store.config.js**. Use the [`metadataPath`](../../submit/eas-json.md#metadatapath) property in the **eas.json** file to pick the JS config file.
```